scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,9 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    brent does not ensure that the located minimum is within the initially
```

Comment:
Though there is still a mystery here--we can do

```python
>>> from scipy.optimize import fminbound
>>> fminbound(lambda x: x**2, -1, 0)
-5.9608609865491405e-06
```

i.e. `fminbound` appears to be correctly getting at the minimum occurring at an endpoint of the interval. You couldn't do that with `brent` *and* a bracketing interval (since one doesn't exist). That could stand to be clarified.